enos/enos.py

```diff
@@ -398,13 +405,17 @@ def cartesian(d):
                     }
 
                     if "plugin" in scenario:
-                        plugin = os.path.abspath(os.path.join(workload_dir, scenario["plugin"]))
+                        plugin = os.path.abspath(
+                                os.path.join(workload_dir, scenario["plugin"]))
```

Comment:
Because this is a PR about code beauty, let me say that this `os.path.join` seems weirdly indented.